internal/hostservice/service.go

```diff
@@ -0,0 +1,41 @@
+package hostservice
+
+import (
+	"github.com/blakewilliams/remote-development-manager/internal/hostservice/clipboard"
+	"github.com/blakewilliams/remote-development-manager/internal/hostservice/open"
+)
+
+// Service is the set of functionalities made available by the server to the client.
+type Service interface {
+	clipboard.Clipboard
+	open.Opener
+}
+
+// HostService is a default implementation of Service which offers copy paste
+// and open functionality.
+type HostService struct {
+	clipboard.Clipboard
+	open.Opener
+}
+
```

Comment:
```suggestion
}

var _ Service = (*HostService)(nil)

```

Thoughts on using the type system to validate `HostService` implements the `Service` interface?

Reply:
Ah cool! I'm unfamiliar with this idiom. Does this just create a compile time assertion that HostService implements Service in case of future regressions?

Reply:
Yep! exactly.